### ISSUE TITLE
Need a method to convert struct to FixedDataGrid type

### ISSUE THREAD

**Issue by rishiloyola** [sjwhitworth/golearn#190]
I could not find a direct way to convert my struct into FixedDataGrid data type to predict and train the model. I have to save them in csv file then only I am able to train my model.

**Comment by Sentimentron** [comment]
Yeah, this could be a bit more streamlined. I wonder if it's possible to use `go generate` to add the appropriate `base.DenseInstances` methods?
